classes/01class/exercises/c01-git03/chisholm-ray/README.md

```diff
@@ -0,0 +1,20 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+The size of the repo doubled once the file was added
+
+2. What is the reason for this problem?
+The size of the repository was double the size of the file since it exists both within the working directory and the git repository.
+Before adding the file to the repo the file only existed in the working directory. 
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+git revert
+
+4. What are the consequences of the previous action?
+when you run 'git revert' a new commit is made that undoes all the changes of the previous commit. The files will remain in the previous commit history and all history is left untouched by 'git revert' as it is not removing anything.
```

Comment:
Yes, `git revert` will have that problem. 
try [git filter-branch](https://git-scm.com/docs/git-filter-branch), but it comes with a [warning](https://git-scm.com/docs/git-filter-branch#_warning). Also, have a look at these [issues](https://github.com/newren/git-filter-repo/#filter-branch) with it as well. 

As git suggests, an alternative tool to use is [git-filter-repo](https://github.com/newren/git-filter-repo/), instead of git filter-branch.